lib/tasks/jobs/migrations.rake

```diff
@@ -1,12 +1,16 @@
 namespace :db do
   namespace :migrations do
```

Comment:
question - why could we not make the namespace more descriptive like `:migrate_again_and_retry` or something along those lines. Maybe it's a bit too wordy.

Reply:
So the "namespace" is a bit generic, like "broadly what is this about". Then line 4, the task name, tries to be a bit descriptive. I thought that "run_with_retry" in the context of "migrations" captured what we wanted here, but yes, you could be as descriptive as you like.